workloads/router-perf-v2/workload.py

```diff
@@ -17,10 +17,17 @@
 number_of_routers = os.getenv("NUMBER_OF_ROUTERS", "")
 
 
-def index_result(payload):
+def index_result(payload, retry_count=3):
     print(f"Indexing documents in {es_index}")
-    es = elasticsearch.Elasticsearch([es_server], send_get_body_as='POST')
-    es.index(index=es_index, body=payload)
+    while retry_count > 0:
+        try:
+            es = elasticsearch.Elasticsearch([es_server], send_get_body_as='POST')
+            es.index(index=es_index, body=payload)
+            retry_count = 0
+        except Exception as e:
```

Comment:
curious, could this ever end up with us re-indexing multiple sets of the same results, in the case the index fails in the middle of the payload being sent?

Reply:
This is not possible since this script only calls mb once (and indexes only one doc) with every iteration/sample.